File: JULIA/Cap05/c05ex02.jl
# c05ex02.jl

global resp = "S"
while (uppercase(resp) == "S")
  print("Entre um valor numerico: ")
  n = parse(Int, readline())

  r = n * 3

  println("Resultado = $r")
  println()

  print("Deseja continuar? (S/N): ")
  global resp = readline()

  println()
end

println()
print("Tecle <Enter> para encerrar... ")
readline()

File: KOTLIN/Cap05/c05ex02.kt
// c05ex02.kt

fun main() {
    var RESP: Char = 'S'
    var N: Int
    var R: Int

    while (RESP == 'S') {
        print("Entre um valor numerico: ")
        N = readLine()?.toInt() ?: 0

        R = N * 3

        println("Resultado = $R\n")

        print("Deseja continuar? (S/N): ")
        RESP = readLine()?.uppercase()?.get(0) ?: 'N'
        
        println()
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
